### PR TITLE
142737153 kh johari poro tests

### PR DESCRIPTION
**Pull request by kheppenstall**
-Refactor how frequencies are calculated in johari pr.
-Add tests for johari PORO.

@Dpalazzari @akintner @lucyconklin @wlffann Ready for review. Let's all review and comment on this one since its pretty big.

@case-eee FYI

### CODE REVIEW

**Comment by Dpalazzari** [comment]
Nice @kheppenstall This still says WIP, can I merge it or are you still doing more on it?

**Comment by kheppenstall** [comment]
@Dpalazzari Lets hold off on this one and talk about it Monday. Not sure if we'll want to actually include it or not. The johari is confusing and I think we were originally calculating the frequencies in a way that may have been confusing.

Not sure though so let's wait on it.

**Comment by Dpalazzari** [comment]
I did notice some big changes you made to our Johari PORO. Were you trying to refactor or did we do it wrong? @kheppenstall

**Comment by akintner** [comment]
I just checked through all of this, and I think it's ready for merge. I'm happy to merge it as soon as @lucyconklin  and @wlffann take a peek, as well.

**Comment by wlffann** [comment]
I'm good with merging this. Thanks for taking on this project, @kheppenstall

File: app/models/johari.rb [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
-      adjective if peer_described.include?(adjective)
-    end.compact
-    create_window(adjectives)
+  def window(adjectives)
+    adjectives.uniq.map { |adj| {'name': adj.name, 'frequency': 1} }
+  end
+
+  def window_with_freq(adjectives)
+    total = peer_described.count.to_f / 10
+    counts = get_counts(adjectives)
```
Comment (lucyconklin) [inline app/models/johari.rb]: Thanks for explaining this part!

**Comment by lucyconklin** [comment]
Looks good to me!

**Pull request merged**